extensions/wikia/InsightsV2/specials/PopularPages.class.php

```diff
@@ -73,12 +73,12 @@ public function recache( $limit = false, $ignoreErrors = true ) {
 	 * @return array
 	 */
 	public function reallyDoQuery( $limit = false, $offset = false ) {
-		global $wgCityId;
+		global $wgCityId, $wgContentNamespaces;
 
 		$topArticles = DataMartService::getTopArticlesByPageview(
 			$wgCityId,
 			null,
-			null,
+			$wgContentNamespaces,
```

Comment:
@rybmat In what case could a user delete NS_MAIN from $wgContentNamespaces? Only staff can edit this variable and I don't think comsup would ever allow a wiki to not have NS_MAIN as content NS